modules/ROOT/pages/faq.adoc

```diff
@@ -3,21 +3,20 @@
 [#faq-q1]
 == How to access the Knative services ?
 
-When looking up the IP address to use for accessing your app, you need to look up the NodePort for the **istio-ingressgateway** well as the IP address used for minishift. You can use the following command to look up the value to use for the pass:[{IP_ADDRESS}] placeholder used in the sample
+When looking up the IP address to use for accessing your app, you need to look up the NodePort for the **istio-ingressgateway** as well as the IP address used for minikube. You can use the following command to look up the value to use for the pass:[{IP_ADDRESS}] placeholder used in the sample
 [source,bash,linenums]
 ----
 #!/bin/bash
 # In Knative 0.2.x and prior versions, the `knative-ingressgateway` service was used instead of `istio-ingressgateway`.
 INGRESSGATEWAY=knative-ingressgateway
 
 # The use of `knative-ingressgateway` is deprecated in Knative v0.3.x.
-# Use `istio-ingressgateway` instead, since `knative-ingressgateway`
-# will be removed in Knative v0.4.
+# Use `istio-ingressgateway` instead.
 if kubectl get configmap config-istio -n knative-serving &> /dev/null; then
     INGRESSGATEWAY=istio-ingressgateway
 fi
 
-IP_ADDRESS="$(minishift ip):$(kubectl get svc $INGRESSGATEWAY --namespace istio-system --output 'jsonpath={.spec.ports[?(@.port==80)].nodePort}')"
+IP_ADDRESS="$(minikube ip):$(kubectl get svc $INGRESSGATEWAY --namespace istio-system --output 'jsonpath={.spec.ports[?(@.port==80)].nodePort}')"
```

Comment:
I think at this point minishift doesn't work with Knative so changed this to minikube. It would be also nice to have the same information for OpenShift where we can call the URL directly and don't have to bother with NodePort (we use LoadBalancer instead)